src/bundle/Resources/public/scss/_mixins.scss

```diff
@@ -172,9 +172,15 @@
         content: ': ';
     }
 
-    &.required {
+    &.required:not(.ez-data-source__label) {
         &:after {
             content: '*: ';
         }
     }
+
+    &.required.ez-data-source__label {
```

Comment:
Cannot we just drop using this mixin for `.ez-data-source__label`? https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/public/scss/fieldType/edit/_base-field.scss#L3

And just add `&:after {content: ": "}` https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/public/scss/fieldType/edit/_base-field.scss#L51 or in a twig?

Because for me it seems like generic mixin gets information about where is used and it's no longer generic. ;)

Reply:
Thought about this, but then it needs to be somehow duplicated in 
https://github.com/ezsystems/ezplatform-admin-ui/blob/dbffc62c46e8b6137895402184b286fae43f2f97/src/bundle/Resources/public/scss/_forms.scss#L2

I am right?